### ISSUE TITLE
INSTALLATION ERROR: FileNotFoundError: [WinError 2] The system cannot find the file specified

### ISSUE THREAD

**Issue by Moaaz10** [openMVG/openMVG#2287]
I am using windows 10. just finished installing and I am trying out the sample like it says in the doc. I have a problem with the running the sample. feel like its something simple but just can't figure it out

This is the code I ran in the cmd in build\software\SFM dir

E:\openMVG\buildfrfr\software\SfM>py SfM_SequentialPipeline.py E:\openMVG\ImageDataset_SceauxCastle-master\images E:\openMVG\ImageDataset_SceauxCastle-master\result

This is the result: 
Using input dir  :  E:\openMVG\ImageDataset_SceauxCastle-master\images
      output_dir :  E:\openMVG\ImageDataset_SceauxCastle-master\result
1. Intrinsics analysis
Traceback (most recent call last):
  File "E:\openMVG\buildfrfr\software\SfM\SfM_SequentialPipeline.py", line 50, in <module>
    pIntrisics = subprocess.Popen( [os.path.join(OPENMVG_SFM_BIN, "openMVG_main_SfMInit_ImageListing"),  "-i", input_dir, "-o", matches_dir, "-d", camera_file_params] )
                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\programs\Python\Lib\subprocess.py", line 1026, in __init__
    self._execute_child(args, executable, preexec_fn, close_fds,
  File "D:\programs\Python\Lib\subprocess.py", line 1538, in _execute_child
    hp, ht, pid, tid = _winapi.CreateProcess(executable, args,
                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [WinError 2] The system cannot find the file specified

After a lot of research I found out it has to do with subprocess.Popen() where the arg is not an executable (this is a windows problem apparently) and I should add "shell=True" in the subprocess.Popen in SFM_sequentialpiepline.py

This is the result after adding the "shell=True":

Using input dir  :  E:\openMVG\ImageDataset_SceauxCastle-master\images
      output_dir :  E:\openMVG\ImageDataset_SceauxCastle-master\result
1. Intrinsics analysis
The system cannot find the path specified.
2. Compute features
The system cannot find the path specified.
3. Compute matching pairs
The system cannot find the path specified.
4. Compute matches
The system cannot find the path specified.
5. Filter matches
The system cannot find the path specified.
6. Do Sequential/Incremental reconstruction
The system cannot find the path specified.
7. Colorize Structure
The system cannot find the path specified.
 

what should I do? .-.

**Comment by pmoulon** [comment]
Did you solve your issue? Sometime the path of where the binaries are compiled is not right depending of your compilation settings.

**Comment by Moaaz10** [comment]
Thank you for the reply! Yes I solved it. I wanted to make sure everything is working correctly before I commented on what happened. 
My VS for some odd reason kept turning the build option from "release" to "debug", so it couldn't find the file after build. 
For people who have similar problems make sure this doesn't flip back to debug before building:
![Screenshot_4](https://github.com/openMVG/openMVG/assets/44412405/329aa115-ca1b-4bc8-9915-893b05a1a8b3)
